packages/cpt-ui/components/EpsTabs.tsx

```diff
@@ -0,0 +1,40 @@
+'use client'
+import React from 'react'
+import "../assets/styles/tabs.scss"
+import { Tabs } from "nhsuk-react-components";
+import PrescriptionIdSearch from "./prescriptionSearch/PrescriptionIdSearch"
+import NhsNumSearch from './prescriptionSearch/NhsNumSearch';
```

Comment:
These components seem to use hard-coded English text - do they need to be parameterised?

Reply:
These components are only placeholders for the yet to be dev'ed search pages, so they are pretty much an MVP.  The actual pages will follow the standard practise when developed